src/Tribe/Event_Status/Event_Status_Provider.php

```diff
@@ -306,4 +316,30 @@ public function filter_include_single_status_reason( $notices_html, $notices ) {
 	public function filter_insert_status_label( $hook_name, $entry_point_name, $template ) {
 		return $this->container->make( Template_Modifications::class )->insert_status_label( $hook_name, $entry_point_name, $template );
 	}
+
+	/**
+	 * Handles the compatibility with the "The Events Calendar Extension: Events Control" plugin.
+	 *
+	 * @since TBD
+	 */
+	public function handle_events_control_extension() {
+		if ( ! class_exists( 'Tribe__Events__Filterbar__View' ) ) {
```

Comment:
We should either move this to Filterbar or create a single method to test this. Don't use class exist, it's better to use a Constant check.

Reply:
Is this the correct class here?
The function is `handle_events_control_extension` but we're checking for FBAR?

Reply:
Changed the check to the extension.